app/scss/layout/_main.scss

```diff
@@ -0,0 +1,36 @@
+.page {
+  background-repeat: no-repeat;
```

Comment:
Main doesn't exist anymore. Everything here is a repetition. .main is already .section--hero and .page is in _page.scss